### PR TITLE
Rework Liquid filters documentation

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors Liscense Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the chart below to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub)
- [Technical Documentation Style Guide](https://docs.google.com/document/d/e/2PACX-1vTtzHpcihaXTTYD85LoKIvYBvpCQFLr8n0BDKRDRAMEz_DnZdHJJINKL24r4JXkRUui24pl_DVxbu2T/pub#h.wstt3flbts5k)
- [Styling Test Page](https://github.com/braze-inc/braze-docs/blob/develop/_docs/_home/styling_test_page.md)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then `Docs`, this can take up to 5 minutes to load. A `502` error means the deployment is still loading, refresh the page, and try again.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Application Infrastructure<br>Data Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Intelligence<br>Product Partnerships<br>SMS<br>Internal Tools</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Ingestion<br>Platforms and Channels<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Messaging and Automation<br>Dashboard Infrastructure<br>Email</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md [hunk 1]
```diff
@@ -6,151 +6,424 @@ description: "Filters can be used to reformat static or dynamic content. This re
...
+| [modulo][4.11]    | Divides an output by a number and returns the remainder.                                                                       | ✅  Yes   |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
+
+{% alert tip %}
+When dividing integers (whole numbers) by integers in Liquid, if the answer is a float (number with a decimal), Liquid will automatically round down to the nearest integer. However, dividing integers by floats will always give you a float. That means you can turn your integers into a float (1.0, 2.0, 3.0) to return a float.
+{% raw %}
+<br><br>For example,`{{15 | divided_by: 2}}` will output `7`, whereas  `{{15 | divided_by: 2.0}}` will output `7.5`.
+{% endraw %}
+{% endalert %}
+
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:113]: A kind of lengthy example, I might put a new sub heading here like "Mathematical Operations with Custom Attributes"

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md [hunk 2]
```diff
@@ -6,151 +6,424 @@ description: "Filters can be used to reformat static or dynamic content. This re
...
+| :---------------------------------- | :--------------------------------------------------------------------------------------------------------------------- | :-------- |
+| [money][5.1]                        | Formats numbers to ensure that decimals are in the proper place, and zeros are not dropped off the end of any numbers. | ✅  Yes   |
+| [money_with_currency][5.2]          | Formats numbers with the currency symbol. | ✅  Yes   |
+| [money_without_trailing_zeros][5.3] | Formats numbers to exclude the decimal separator (either `.` or `,`) and trailing zeros. If there are no trailing zeros, then this filter behaves like the `money` filter. | ✅  Yes   |
+| [money_without_currency][5.4]       | Formats numbers without the currency symbol.                                                       | ⛔  No    |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
 
-If you're updating a user on their purchase, an account balance, or anything regarding money, you can use money filters to ensure that your decimals are in the proper place and that no piece of your update is lost (like that pesky `0` at the end).
+{% alert warning %}
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:165]: Would also make a sub header here so people can reference these examples quickly, maybe "Shopify money filter vs Braze money filter", or maybe "money filter best practices"?

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md [hunk 3]
```diff
@@ -6,151 +6,424 @@ description: "Filters can be used to reformat static or dynamic content. This re
...
 ```liquid
-2016-08-22 18:13:13
+2021-08-04 18:13:13
 ```
-
 {% endraw %}
+{% endtab %}
+{% endtabs %}
+
+
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:360]: Maybe note that there are advanced filters outside of these filters that the customer can use. (link out to advanced filter documentation)

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md [hunk 4]
```diff
@@ -6,151 +6,424 @@ description: "Filters can be used to reformat static or dynamic content. This re
...
 ```
+{% endraw %}
+{% endtab %}
+{% endtabs %}
 
+## Money Filters
 
+If you’re updating a user on their purchase, an account balance, or anything regarding money, you should use money filters. Money filters ensure that your decimals are in the proper place and that no piece of your update is lost (like that pesky `0` at the end).
 
-## E-Commerce
```
Comment (KellieHawks) [inline _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:68]: needs redirect from old ecomerce link to redirect tomoney filter link

File: _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md (hunk 3 above)
Comment (KellieHawks) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:360]: or maybe even reference the liquid use case library somewhere in the doc

Comment (bre-fitzgerald) [reply _docs/_user_guide/personalization_and_dynamic_content/liquid/filters.md:366]: Added both!

**Review by KellieHawks — APPROVED** [review]
Lgtm!

**Pull request merged**